email_scraper/src/test/java/EmailScraperTest.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class EmailScraperTest {
```

Comment:
you have this test code here that looks like gradle made for you, but it doesn't compile. did you think about writing any tests? or alternatively, might you have simply left this out rather than check in the not-compiling code?

Reply:
You are right. I did not spend time on creating meaningful unit test cases due to extremely tight time schedule that I was trapped in... I forgot to fix the testing code after I refactored the main code. The wrong code should definitely not be checked in. I fixed the testing code, and it works now. 

FYI: Regarding to adding meaningful unit test cases, my original plan was to use mockito to mock jsoup to unit test my code.